### ISSUE TITLE
Catalogue different types of VITS signals for auto-tuning

### ISSUE THREAD

**Issue by happycube** [happycube/ld-decode#490]
A long-deferred (and possibly longer-deferred) project of mine is to auto-tune not just black/white levels, but color levels and phase.  I'd like to build a set of samples of each common type of VITS for both PAL and NTSC over time.

(After #481 NTSC auto chroma gain was removed, but I'd like to reinstate - and improve - it... the burst level even with auto-MTF changes especially later in CAV disks.  I'll hopefully have examples with 12" GGV soon.)

**Comment by toynbeeidea** [comment]
I don't know how/where to obtain VITS signals, but I have a lot of NTSC discs and am willing to provide... something?  I guess I need some instructions.

**Comment by simoninns** [comment]
no help in 5 years - close wontfix
